tracks/ruby/exercises/word-count/mentoring.md

```diff
@@ -1,5 +1,5 @@
 ### New Concepts
-hash (new here because Robot Name has been removed as a core exercise); constant; `each_with_object`.
+Hash (it's new here because Robot Name has been removed as a core exercise); constant; `each_with_object`: avoiding the Accumulator Pattern.
 
```

Comment:
Good question. I'm not sure - maybe "avoiding the explicit Accumulator Pattern"?
Curious to hear other opinions.

Reply:
@kotp What do you think?

Reply:
The iterator takes care of the accumulation for you, so you aren't doing something like:

```ruby
accumulation = 0
collection.each { |e| accumulation += 1 if e }
accumulation.max
```

Where we are handling the details of that accumulator ourselves, by initializing it, having it be built up and then working on that accumulation later. (or returning it explicitly).

It is that pattern (not accumulation) that we are suggesting to avoid.

Reply:
I rephrased this.

Reply:
I think that works. Another suggestion in case you like it better: "avoiding manual accumulation".